### ISSUE TITLE
Disable pusher cache to get START_ROOM_URL's room

### ISSUE THREAD

**Issue by sebght** [thecodingmachine/workadventure#1929]
Requirements :
- WorkAdventure 1.7.1
- Chrome 98.0.4758.102 (latest or close)

How to replicate :
- setup your own workadventure self-hosted server
- setup the variable `START_ROOM_URL` with the url of your landing map
- `docker-compose up`
- `docker-compose down`
- change the `START_ROOM_URL` with another map url
- `docker-compose up`

What happens : the user is redirected to the first map

What the browser is doing : there is `lastRoomUrl` variable in the browser local storage, which is used by the service to resolve which url it must get, before asking for `START_ROOM_URL` env variable.

Would it be possible to remove this cache functionality ?

**Comment by tabascoeye** [comment]
I have to agree that it is pretty annoying sometimes that lastRoomUrl is just used.
Maybe we can make it an Option like 
"[  ] Save the map you are currently on"

comments?

**Comment by ValdoTR** [comment]
@tabascoeye Yes, an option to "remember the last room visited" would be nice
If you want to create an issue or a PR ;)
